sdk/tables/tests.yml

```diff
@@ -12,6 +12,24 @@ stages:
         AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        STORAGE_ACCOUNT_NAME: $(python-storage-storage-account-name)
+        STORAGE_ACCOUNT_KEY: $(python-storage-storage-account-key)
+        STORAGE_DATA_LAKE_ACCOUNT_NAME: $(python-storage-data-lake-account-name)
+        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(python-storage-data-lake-account-key)
+        BLOB_STORAGE_ACCOUNT_NAME: $(python-storage-blob-storage-account-name)
+        BLOB_STORAGE_ACCOUNT_KEY: $(python-storage-blob-storage-account-key)
+        REMOTE_STORAGE_ACCOUNT_NAME: $(python-storage-remote-storage-account-name)
+        REMOTE_STORAGE_ACCOUNT_KEY: $(python-storage-remote-storage-account-key)
+        PREMIUM_STORAGE_ACCOUNT_NAME: $(python-storage-premium-storage-account-name)
+        PREMIUM_STORAGE_ACCOUNT_KEY: $(python-storage-premium-storage-account-key)
```

Comment:
Surely we don't need premium storage or remote storage or datalake?

Reply:
You're right, I was copied from storage tests.yml to try to resolve the environment variables missing issue. Will do some cleanup.